conf/modules.config

```diff
@@ -192,14 +197,18 @@ process {
         withName: OPENMS_MZTABEXPORTERPSM {
                 ext.prefix  = { "${meta.sample}_${meta.condition}_all_ids_merged" }
                 publishDir  = [
-                    path: { "${params.outdir}/Intermediate_Results" },
+                    path: { "${params.outdir}/intermediate_results/refined_fdr" },
                     mode: params.publish_dir_mode,
                     pattern: '*.mzTab'
                 ]
             }
 
         withName: MHCFLURRY_PREDICTPSMS {
```

Comment:
Name is a little bit ambigous imo. We don't predict PSMs, we predict the peptides resulting from the peptide spectrum match. MHCFLURRY_PREDICT_PEPTIDES makes more sense to me

Reply:
I do not understand what you want to mention here regarding the naming @jonasscheid. About the order of the process I would be open for suggestions

Reply:
No I think it is fine like that regarding the naming. Now I understood it better!